README.md

```diff
@@ -3,42 +3,49 @@
+OpenDP is not yet recommended for production deployments. It has known privacy concerns, particularly around floating point handling and side channel attacks. We expect to address these issues, but until then, *please don't use OpenDP for any privacy-critical applications*.
 
-## Status
-OpenDP Core is early in its initial development. We're building out the main concepts of the OpenDP Programming Framework, and providing implementations of several common privacy mechanisms and summary statistics. Our focus is on correctness and developer usability.
+More details can be found in the [Limitations section of the User Guide](https://docs.opendp.org/en/stable/user/limitations.html)
 
-The current code is functional, but not ready for general usage. The APIs are unstable and subject to extensive change. Aspects of the library are likely difficult to understand without examining the sources. Documentation and sample code is forthcoming.
+## Installation
 
-## Communication
-- You are very welcome to join us on [GitHub Discussions](https://github.com/opendp/opendp/discussions)!
-- Please use [GitHub Issues](https://github.com/opendp/opendp/issues) for bug reports and feature requests.
-- For other requests, including security issues, please contact us at [info@opendp.org](mailto:info@opendp.org).
+The easiest way to install OpenDP is using `pip` (the [package installer for Python](https://pypi.org/project/pip/)):
 
+    % pip install opendp
 
-## Install
-- [From PyPI](https://pypi.org/project/opendp/)
-- [From Crates.io](https://crates.io/crates/opendp)
-- [From Source](https://docs.opendp.org/en/latest/resources/dev-guide/general-logistics/dev-environment.html)
+More information can be found in the [Getting Started section of the User Guide](https://docs.opendp.org/en/stable/user/getting-started.html)
 
 ## Documentation
-- [Overview](https://docs.opendp.org)
-- [Python API](https://docs.opendp.org/en/latest/api/python/index.html)
-- [Rust API](https://docs.rs/opendp/)
-    
\ No newline at end of file
+
+The full documentation for OpenDP is located at https://docs.opendp.org. Here are some helpful entry points:
+
+* [User Guide](https://docs.opendp.org/en/stable/user/index.html)
+* [Python API Docs](https://docs.opendp.org/en/stable/api/python/index.html)
+* [Developer Guide](https://docs.opendp.org/en/stable/developer/index.html)
+
+## Getting Help
+
+If you're having problems using OpenDP, or want to submit feedback, please reach out! Here are some ways to contact us:
+
+* Ask questions on our [discussions forum](https://github.com/opendp/opendp/discussions)
+* Open issues on our [issue tracker](https://github.com/opendp/opendp/issues)
+* Send critical bugs to [security@opendp.org](mailto:security@opendp.org)
```

Comment:
Ah, sorry, security@opendp.org is not operational yet. We're tracking it in #241. For now security issues should go to info@ like it says on the the Contact page in the docs.

Reply:
Ah ok good to know.